Source/Spring2022_Capstone/Enemies/BaseEnemy.cpp

```diff
@@ -0,0 +1,46 @@
+// Created by Spring2022_Capstone team
+
+#include "BaseEnemy.h"
+#include "Spring2022_Capstone/HealthComponent.h"
+
+// Sets default values
+ABaseEnemy::ABaseEnemy()
+{
+ 	// Set this character to call Tick() every frame.  You can turn this off to improve performance if you don't need it.
+	PrimaryActorTick.bCanEverTick = true;
+
+	HealthComponent = CreateDefaultSubobject<UHealthComponent>(TEXT("HealthComponent"));
+
+	WeaponMesh = CreateDefaultSubobject<UStaticMeshComponent>(TEXT("WeaponMesh"));
```

Comment:
This might not be necessary assuming enemies are made/animated with their weapons already.